go/service/teams.go

```diff
@@ -245,25 +246,26 @@ func (h *TeamsHandler) TeamAddMember(ctx context.Context, arg keybase1.TeamAddMe
 		return res, err
 	}
 
+	assertion := arg.Username
 	if arg.Email != "" || arg.Phone != "" {
-		isEmail := arg.Email != ""
-		typ := "email"
-		name := arg.Email
-		if !isEmail {
-			typ = "phone"
-			name = arg.Phone
+		var assertionURL libkb.AssertionURL
+		var err error
+		if arg.Email != "" {
+			assertionURL, err = libkb.ParseAssertionURLKeyValue(externals.MakeStaticAssertionContext(ctx), "email", arg.Email, true)
+		} else {
+			assertionURL, err = libkb.ParseAssertionURLKeyValue(externals.MakeStaticAssertionContext(ctx), "phone", arg.Phone, true)
 		}
-
-		if err := teams.InviteEmailPhoneMember(ctx, h.G().ExternalG(), arg.TeamID, name, typ, arg.Role); err != nil {
-			return keybase1.TeamAddMemberResult{}, err
+		if err != nil {
+			return res, err
 		}
-		return keybase1.TeamAddMemberResult{Invited: true, EmailSent: isEmail}, nil
```

Comment:
looks like this was the only place where `EmailSent` was being set, maybe we should remove it completely? unless my editor is being confused, it's also not read anywhere

Reply:
sgtm